unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDController.cs

```diff
@@ -463,7 +447,7 @@ private void DisplayMoreFriendRequests()
         
         friendsController.GetFriendRequests(
             LOAD_FRIENDS_ON_DEMAND_COUNT, lastSkipForFriendRequests,
-            LOAD_FRIENDS_ON_DEMAND_COUNT, lastSkipForFriendRequests);
+            LOAD_FRIENDS_ON_DEMAND_COUNT, lastSkipForFriendRequests).Forget();
```

Comment:
you could await for the result and then add the resultant requests

Reply:
It is because this logic has been moved directly to `AddFriendRequests(...)` that is called after `await friendsController.GetFriendRequests(...)` and process the results. Now the friend requests list is filled using these results directly, instead of doing it managing the friendship status events.

Reply:
Yeah, with my last refactor, I'm doing exactly that 👍  Please, take a look!